src/coreclr/gc/gc.cpp

```diff
@@ -29192,18 +29194,14 @@ void gc_heap::mark_phase (int condemned_gen_number, BOOL mark_only_p)
             }
         }
 
-#ifdef FEATURE_BASICFREEZE
-#ifdef USE_REGIONS
-        assert (!ro_segments_in_range);
```

Comment:
assume removal of this assert is intentional?

Reply:
yes `ro_segments_in_range` is not defined unless FEATURE_BASICFREEZE is defined.